toast-tk-interpret/src/main/java/io/toast/tk/action/interpret/web/AbstractInterpretationProvider.java

```diff
@@ -4,6 +4,8 @@
 import java.util.List;
 import java.util.Objects;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.Objects'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.talanlabs.com/coding_rules#rule_key=squid%3AUselessImportCheck)